From penberg@cs.helsinki.fi  Mon Oct 27 11:26:04 2008
From: Pekka J Enberg <penberg@cs.helsinki.fi>
Date: Wed, 22 Oct 2008 19:05:54 +0300 (EEST)
Subject: Staging: w35und: make wb35_probe() and wb35_disconnect() funtions static
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: Pavel Machek <pavel@suse.cz>, linux-kernel@vger.kernel.org
Message-ID: <Pine.LNX.4.64.0810221905180.1679@melkki.cs.Helsinki.FI>


From: Pekka Enberg <penberg@cs.helsinki.fi>

The wb35_probe() and wb35_disconnect() functions are only used in wbusb.c so
make them static and remove them from a header file.

Signed-off-by: Pekka Enberg <penberg@cs.helsinki.fi>
Acked-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/linux/wbusb.c   |    6 ++----
 drivers/staging/winbond/linux/wbusb_f.h |    4 ----
 2 files changed, 2 insertions(+), 8 deletions(-)

--- a/drivers/staging/winbond/linux/wbusb.c
+++ b/drivers/staging/winbond/linux/wbusb.c
@@ -185,9 +185,7 @@ static const struct ieee80211_ops wbsoft
 struct wbsoft_priv {
 };
 
-
-// Usb kernel subsystem will call this function when a new device is plugged into.
-int wb35_probe(struct usb_interface *intf, const struct usb_device_id *id_table)
+static int wb35_probe(struct usb_interface *intf, const struct usb_device_id *id_table)
 {
 	PADAPTER	Adapter;
 	PWBLINUX	pWbLinux;
@@ -357,7 +355,7 @@ int wb35_close(struct net_device *netdev
 	return 0;
 }
 
-void wb35_disconnect(struct usb_interface *intf)
+static void wb35_disconnect(struct usb_interface *intf)
 {
 	PWBLINUX pWbLinux;
 	PADAPTER Adapter = usb_get_intfdata(intf);
--- a/drivers/staging/winbond/linux/wbusb_f.h
+++ b/drivers/staging/winbond/linux/wbusb_f.h
@@ -18,10 +18,6 @@ void WbUsb_destroy(phw_data_t pHwData);
 unsigned char WbWLanInitialize(PADAPTER Adapter);
 #define	WbUsb_Stop( _A )
 
-int wb35_probe(struct usb_interface *intf,const struct usb_device_id *id_table);
-void wb35_disconnect(struct usb_interface *intf);
-
-
 #define wb_usb_submit_urb(_A) usb_submit_urb(_A, GFP_ATOMIC)
 #define wb_usb_alloc_urb(_A) usb_alloc_urb(_A, GFP_ATOMIC)
 
